playbooks/common/openshift-cluster/initialize_facts.yml

```diff
@@ -136,6 +136,21 @@
         sdn_network_plugin_name: "{{ os_sdn_network_plugin_name | default(None) }}"
         use_openshift_sdn: "{{ openshift_use_openshift_sdn | default(None) }}"
 
+  - name: Default system_images_registry to a enterprise registry
+    set_fact:
+      system_images_registry: "registry.access.redhat.com"
+    when:
+    - system_images_registry == ""
```

Comment:
` - system_images_registry | default('') == ''` in case it's not set or do you default this elsewhere?